wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -26,6 +26,8 @@ public typealias SelfUserType = UserType & SelfLegalHoldSubject
 /// layer.
 public protocol UserSession: AnyObject {
 
+    var useCaseFactory: UseCaseFactoryProtocol { get }
```

Comment:
Does this need to be exposed? we could (like we already do) expose protocol requirements to get the use case, but internal the implementation can use the use case factory.

Reply:
Fixed it!